README.rst

```diff
@@ -15,12 +15,14 @@ Setup
 
 Start by `creating a project in the Google Developers Console <https://console.developers.google.com>`_.
 
+In the **Authorized JavaScript origins** add the Root URL for your installation::
```

Comment:
was this actually needed? its unclear why since we dont do any JS calls to Google